Build fails in screen-shot-callback.hpp:64:28 with `runtime_error` is not a member of `std`

File bigger/include/screen-shot-callback.hpp is missing an inclusion to `<stdexcept>`.

I'm building it with:

1. mkdir build && cd build
2. cmake ..
3. make

Build fails with:

```
In file included from <HOME DIR>/GameDev/bigger/bigger/src/app.cpp:4:
<HOME DIR>/GameDev/bigger/bigger/include/bigger/screen-shot-callback.hpp: In member function ‘virtual void bigger::ScreenShotCallback::screenShot(const char*, uint32_t, uint32_t, uint32_t, const void*, uint32_t, bool)’:
<HOME DIR>/GameDev/bigger/bigger/include/bigger/screen-shot-callback.hpp:65:28: error: ‘runtime_error’ is not a member of ‘std’
   65 |                 throw std::runtime_error("");
      |                            ^~~~~~~~~~~~~
make[2]: *** [CMakeFiles/bigger.dir/build.make:162: CMakeFiles/bigger.dir/src/app.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:361: CMakeFiles/bigger.dir/all] Error 2
make: *** [Makefile:149: all] Error 2
```

Adding an `#include <stdexcept>` fixes the issue.

Thank you very much!

--- a/include/bigger/screen-shot-callback.hpp
+++ b/include/bigger/screen-shot-callback.hpp
@@ -4,6 +4,7 @@
 #include <bgfx/bgfx.h>
 #include <bimg/bimg.h>
 #include <bx/file.h>
+#include <stdexcept>
 
 namespace bigger
 {

